### DISCUSSION TITLE
how to use  "Sampling"?

### DISCUSSION THREAD

**Discussion by Leona-kk** [CURENT/andes#466] [Q&A]
Hi Dr.Cui and other developers,

In the Discrete section, I notice that the sampling class can be used to implement a 4-second sampling of the AGC signal.  But how it works? Do we have any examples for this class? 

Thank you.

**Comment by jinningwang** [comment]
Here is an example model that uses this block, https://github.com/CURENT/andes/blob/d20c689f8b98195c75327a315ffca0ead3353272/andes/models/area.py#L136
